regression/cbmc-shadow-memory/no-shadow-memory-matching1/test.desc

```diff
@@ -1,4 +1,4 @@
-FUTURE
+CORE
 main.c
 --no-shadow-memory-matching '.*\.c:.*__cs_.*'
```

Comment:
Flag doesn't exist yet, should be left `FUTURE` now.

Reply:
Removed